openupgrade_scripts/scripts/product/18.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,109 @@
+---Models in module 'product'---
+new model product.combo
+new model product.combo.item
+
+# DONE: rename models and fields from point_of_sale models if installed
```

Comment:
All these references miss the usual format `# DONE: pre-migration:` or `# DONE: post-migration:` to indicate where this is done.

And we don't usually put an empty line in between for being more easily linkable to the modifications block.